index.html

```diff
@@ -1,11 +1,45 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,800,600,300,300italic' rel='stylesheet' type='text/css'>
     <title>Relaxr</title>
     <link rel="stylesheet" type="text/css" href="css/style.css">
   </head>
   <body>
-    <header>
-    </header>
+  	<header><div class="bg1">
+	    <h1>
+	    Relaxr
+	    </h1>
+
+	    <h2>Get piece of mind with a single tap</h2>
+	     <a class="button" href="#">Get it Now</a>
+     </div></header>
+     <div class="bg2">
+        <div class="box">
+     	<h3> Benefits</h3>
+     		<p> The perfect personal assistant, Relaxr does your job for you so you can enjoy life as it is meant to be.</p>
+     		<ul>
+     			<li>Schedule meetings for you</li>
+     			<li>Excel automation to complete your work for you</li>
+     			<li>Responds to emails on your behalf</li>
+     			<li>Does all your work for you with your revolutionary AI technology</li>
+     		</ul>
+        </div>
+     </div>
+     <footer><div class="bg3">
```

Comment:
I think the footer starts after the Sign Up Now button below. I would move this after that div.